packages/feedback/src/sass/component/_component.sass

```diff
@@ -0,0 +1,537 @@
+@import "../colors/_colors.sass"
+@import "../mixin/_mixin.sass"
+
+@import "~bootstrap/scss/bootstrap.scss"
+
```

Comment:
Why did you copy the entire `_component.sass` file here? You should have just copied the components that you really need.

Reply:
done